### PR TITLE
docs: update fields as they are optional

### PR DESCRIPTION
**Pull request by sicoyle**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Docs updates given these changes
https://github.com/dapr/components-contrib/pull/3562/files

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/postgresql.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ The AWS authentication token will be dynamically rotated before it's expiration
 | `useAWSIAM` | Y | Must be set to `true` to enable the component to retrieve access tokens from AWS IAM. This authentication method only works with AWS Relational Database Service for PostgreSQL databases. | `"true"` |
 | `connectionString` | Y | The connection string for the PostgreSQL database.<br>This must contain an already existing user, which corresponds to the name of the user created inside PostgreSQL that maps to the AWS IAM policy. This connection string should not contain any password. Note that the database name field is denoted by dbname with AWS. | `"host=mydb.postgres.database.aws.com user=myapplication port=5432 dbname=my_db sslmode=require"`|
 | `awsRegion` | Y | The AWS Region where the AWS Relational Database Service is deployed to. | `"us-east-1"` |
-| `awsAccessKey` | Y | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
-| `awsSecretKey` | Y | The secret key associated with the access key | `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"` |
+| `awsAccessKey` | N | AWS access key associated with an IAM account | `"AKIAIOSFODNN7EXAMPLE"` |
```
Comment (JoshVanL) [inline daprdocs/content/en/reference/components-reference/supported-bindings/postgresql.md:68]: Please target the v1.15 branch

**Review by JoshVanL — CHANGES_REQUESTED** [review]

**Comment by hhunter-ms** [comment]
@sicoyle does this PR need to point to 1.15?

**Comment by msfussell** [comment]
@sicoyle - You will need to create a new PR for this

**Comment by sicoyle** [comment]
closing as i am fixing in a diff pr

**Pull request closed without merging**